ckanext/datastore_profiler/tests/test_stringstatistics.py

```diff
@@ -1,62 +1,220 @@
+"""
+Test module for stringsatistics module using pytest
+
+Test following methods under StringStatistics class:
+- unique_count
+- mask_count
+- word_count
+
+Contain following functions:
+- test_null_input_error
+- test_string_lenght_min
+- test_string_lenght_max
+- test_all_unique
+- test_all_numeric
+- test_unique_string_counts
+- test_unique_mask_counts
+- test_unique_word_counts
+- test_word_stats
+"""
+
 # User defined modules
 from utils.stringstatistics import StringStatistics
+import pytest
+
+test_data = [
+    "val2",
+    "val3",
+    "val4",
+    "ccc-9",
+    "d-1",
+    "val-3",
+    "val-3",
+    "val1 val2 val3",
+    "this is val1",
+    "this is val2",
+    None,
+    None,
+]
+
+
+def test_null_input_error():
+    """
+    Test whether ValueError will be raised in case of empty or only None
+    input into StringStatistics().unique_count method.
+    """
+
+    test_null_input = [None, None, None]
+    test_empty_input = []
+
+    with pytest.raises(ValueError):
+        StringStatistics().unique_count(test_null_input)
+        StringStatistics().unique_count(test_empty_input)
+
+
+def test_string_lenght_min():
+    """
+    Test lenght of string with minimum lenght will be returned correctly
+    """
+
+    # get stats from appropriate class
+    dict_stats = StringStatistics().unique_count(test_data)["min_string_length"]
+
+    # set the expected results
+    correct_stats = 3
+
+    # assert response
+    assert dict_stats == correct_stats
+
+
+def test_string_lenght_max():
```

Comment:
Im never sure if I should call out typos or not...

... but there's a typo on `length` here 😅